ui/document_uploader.py

```diff
@@ -564,72 +542,91 @@ def upload_scanned_certs(self, scanned_certs):
 
                 new_filename = ''
                 if not 'hashed_filename' in support_doc:
-                    doc_copy_error = doc_copy_error + ' Key hashed_filename'
+                    doc_copy_error = (doc_copy_error
+                                     + ' Key hashed_filename')
                 else:
                     new_filename = support_doc['hashed_filename']
 
                 doc_copy_result = False
                 if doc_copy_error == '':
                     doc_type_cache[src_doc_type_id] = doc_type
-                    doc_copy_result = self.create_new_support_doc_file(
-                        scanned_cert,
-                        new_filename,
-                        doc_type
-                    )
+                    doc_copy_result =\
+                        self.create_new_support_doc_file(
+                            scanned_cert,
+                            new_filename,
+                            doc_type
+                        )
 
                 if not doc_copy_result:
-                    msg = _translate(
-                        'DocumentUploader',
-                        u'Error creating file: {0}',
-                        None
-                        ).format( '`' + new_filename + '`')
-                    self.upload_progress.emit(UploadWorker.ERROR, msg)
+                    msg = self.tr(
+                        u'Error creating file: {0}'.format(
+                            '`' + new_filename + '`'
+                        )
+                    )
+                    self.upload_progress.emit(
+                        UploadWorker.ERROR,
+                        msg
+                    )
                     continue
 
-                msg = _translate(
-                    'DocumentUploader',
-                    u'Finished creating file: {0}',
-                    None
-                    ).format( '`' + new_filename + '`')
-                self.upload_progress.emit(UploadWorker.INFORMATION, msg)
+                msg = self.tr(
+                    u'Finished creating file: {0}'.format(
+                        '`' + new_filename + '`'
+                    )
+                )
+                self.upload_progress.emit(
+                    UploadWorker.INFORMATION,
+                    msg
+                )
                 #Delete After Upload
                 if self.del_after_upload:
                     try:
                         old_filename=scanned_cert['full_filename']
                         os.remove(old_filename)
-                        msg = "Removed file: `"+old_filename+"`"
-                        self.upload_progress.emit(UploadWorker.INFORMATION, msg)
+                        msg = 'Removed file: `'+ old_filename + '`'
+                        self.upload_progress.emit(
+                            UploadWorker.INFORMATION,
+                            msg
+                        )
                     except:

```

Comment:
We need to catch a specific exception e.g. IOError

Reply:
Done @pgathogo